src/geocoder/TECH_DEBT__TemporarilyCopied__Geocoder.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Web;
+using GovUk.Education.SearchAndCompare.Domain.Models;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace GovUk.Education.SearchAndCompare.Geocoder
+{
+
+    /// <summary>
+    ///    Tech debt: this is copy-and-pasted from Geocoder.cs in search-and-compare-ui.
+    ///    We should extract a utilities project these two can share.
+    ///    https://trello.com/c/66mYU1OF/308-tech-debt-extract-geocoder-into-shared-utilities-nuget-package
+    /// </summary> 
+    public class TECH_DEBT__TemporarilyCopied__Geocoder
+    {
+        private readonly string apiKey;
+
+        public TECH_DEBT__TemporarilyCopied__Geocoder(string apiKey)
+        {
+            this.apiKey = apiKey;
+        }
+
+        public async Task<Coordinates> ResolvePostCodeAsync(string postCode)
+        {
+            using(var client = new HttpClient())
```

Comment:
Did you following the wrong bit in 
https://aspnetmonsters.com/2016/08/2016-08-27-httpclientwrong/

Use a single http client please

HttpClient is intended to be instantiated once and reused throughout the life of an application.

Your usage will exhaust the sockets, amongst other things